tests/eas/acceptance.py

```diff
@@ -160,7 +160,24 @@ class SmallTaskPacking(EasTest):
     All tasks run on little cpus.
     """
 
-    conf_basename = "acceptance_small_task_packing.config"
```

Comment:
I cannot see the removal of the acceptance_small_task_packing.config file in this patch.